boa3_test/tests/examples_tests/test_htlc.py

```diff
@@ -1,477 +1,324 @@
-        test_account1_script_hash = test_account1.script_hash.to_array()
-        test_account2 = self.OTHER_ACCOUNT_2
-        test_account2_script_hash = test_account2.script_hash.to_array()
+        transferred_amount_gas = 100 * 10 ** 8
+        person_a = self.account1.script_hash
+        person_b = self.account2.script_hash
         secret_hash = hash160(b'unit test')
+        htlc = self.contract.hash
 
-        runner.add_gas(self.OWNER.address, self.GAS_TO_DEPLOY)
-        htlc_contract = runner.deploy_contract(path, account=self.OWNER)
-        runner.update_contracts(export_checkpoint=True)
-        htlc_script_hash = htlc_contract.script_hash
+        result, _ = await self.call('atomic_swap',
+                                    [person_a, constants.NEO_SCRIPT, transferred_amount_neo,
+                                     person_b, constants.GAS_SCRIPT, transferred_amount_gas,
+                                     secret_hash],
+                                    return_type=bool,
+                                    signing_accounts=[self.owner])
+        self.assertEqual(True, result)
 
-        runner.add_neo(test_account1.address, transferred_amount_neo)
-        runner.add_gas(test_account2.address, transferred_amount_gas)
-
-        # starting atomic swap
-        runner.run_contract(path, 'atomic_swap',
-                            test_account1_script_hash, constants.NEO_SCRIPT, transferred_amount_neo,
-                            test_account2_script_hash, constants.GAS_SCRIPT, transferred_amount_gas,
-                            secret_hash, account=self.OWNER)
-
-        # saving the balance to compare after the transfer
-        balance_neo_sender_before = runner.call_contract(constants.NEO_SCRIPT, 'balanceOf', test_account1_script_hash)
-        balance_neo_receiver_before = runner.call_contract(constants.NEO_SCRIPT, 'balanceOf', htlc_script_hash)
-
-        # this transfer will be accepted
-        invoke = runner.call_contract(constants.NEO_SCRIPT, 'transfer', test_account1_script_hash,
-                                      htlc_script_hash, transferred_amount_neo, None)
+        # transfer won't be accepted, because amount is wrong
+        with self.assertRaises(boatestcase.AbortException):
+            await self.call('transfer',
+                            [person_a, htlc, transferred_amount_neo + 1, None],
+                            signing_accounts=[self.account1],
+                            return_type=bool,
+                            target_contract=constants.NEO_SCRIPT)
+
+        with self.assertRaises(boatestcase.AbortException):
+            await self.call('transfer',
+                            [person_b, htlc, transferred_amount_gas - 1, None],
+                            signing_accounts=[self.account2],
+                            return_type=bool,
+                            target_contract=constants.GAS_SCRIPT)
+
+        await asyncio.sleep(self.REFUND_WAIT_TIME)
```

Comment:
it seems that all `asyncio.sleep` calls in the test case use the same arguments, so you can move it to a helper method call.
```python
async def wait_refund_time():
    return await asyncio.slepp(self.REFUND_WAIT_TIME)
```